app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,5 @@
+class MoviesController < ApplicationController
+
+  def dashboard
```

Comment:
@Trevorsuter should this be `discover` instead?

Reply:
I thought it might be a part of a movies controller, kind of like `users#dashboard` rather than `dashboard#index`. Wasn't too sure about this one, so I'm definitely open to changing it if there's a better option

Reply:
It should. Good catch, thanks!